Gopkg.lock

```diff
@@ -10,8 +10,8 @@
 [[projects]]
   name = "github.com/Sirupsen/logrus"
   packages = ["."]
-  revision = "99069a2f51322071cea01daf0167e3a31a82575a"
-  source = "https://github.com/ChimeraCoder/logrus.git"
+  revision = "f006c2ac4710855cf0f916dd6b77acf6b048dc6e"
```

Comment:
nit: we don't actually have to pin to a hash here, and it's probably better if we don't,  depending on whether logrus released a new minor release. We can punt on that though.

Reply:
I… don't remember doing anything other than "dep ensure -add …" here…

Reply:
Nevermind, this is in the lock file. Though that means that `logrus` is no longer in the Gopkg.toml, I see? That doesn't seem right?

Reply:
/me waves his hands about his dep problems and can fix this he assumes